spec/support/shared_examples/crud_methods.rb

```diff
@@ -1,6 +1,18 @@
 shared_examples_for "Crud Methods" do
-  it ".create posts to the api and returns the new instance" do
+  it "described_class.create posts to the api and returns the new instance" do
```

Comment:
I didn't - but I also didn't think about how much easier the test name will be to read if I did.
I'll get that changed.